src/components/create-courses/component/create-new-author.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from "react";
+import { v4 as uuidv4 } from 'uuid';
+import { Input, Label, Button, Row, InputGroup } from "reactstrap";
+
+export default function AddNewAuthor({ addAuthorToList }) {
+	//"Add New Author" functionality
+	const [newAuthorName, setNewAuthorName] = useState('');
+
+	function createNewAuthor(){
```

Comment:
Move this func outside the component

Reply:
i can't remove this func outside componen because  she use hook/state from this component